.buildkite/pipeline.ml.yml

```diff
@@ -5,7 +5,7 @@
     - RLLIB_TESTING=1 PYTHON=3.7 ./ci/travis/install-dependencies.sh
     - bazel test --config=ci $(./scripts/bazel_export_options)
       --build_tests_only
-      --test_tag_filters=learning_tests_discrete,-fake_gpus,-torch_only,-tf2_only
+      --test_tag_filters=learning_tests_discrete,-fake_gpus,-torch_only,-tf2_only,-no_tf_static_graph
```

Comment:
I am just curious about these tags. where can I use them?

Reply:
They are defined in the `rllib/BUILD` file. You can make up any tags and filter by these on the BAZEL command line.